<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectWidget</class>
 <widget class="QWidget" name="selectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QCheckBox" name="allCBox">
     <property name="text">
      <string>全选</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="reverseCBox">
     <property name="text">
      <string>反选</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelSelectBtn">
     <property name="text">
      <string>取消选择</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
